components/bridge/ChainPicker.tsx

```diff
@@ -2,38 +2,49 @@ import React, { useState, useEffect } from "react";
 import { Chain, SupportedChain } from "../../types";
 
 import styles from "../../styles/components/bridge/chainpicker.module.css";
-import { ETH_LOGO } from "../../utils/bridge/helpers";
-
-const CHAINS: Chain[] = [
-	{
-		name: "Cennznet",
-		logo: "images/cennz.svg",
-	},
-	{
-		name: "Ethereum",
-		logo: ETH_LOGO,
-	},
-];
+import { CHAINS } from "../../utils/bridge/helpers";
 
 const ChainPicker: React.FC<{
 	setChain: Function;
+	setOppositeChain: Function;
 	initialChain: SupportedChain;
+	forceChain: SupportedChain;
 	topText?: string;
-}> = ({ setChain, initialChain, topText }) => {
-	const [chains, setChains] = useState<Chain[]>(CHAINS);
+}> = ({ setChain, initialChain, setOppositeChain, forceChain, topText }) => {
+	const [chains, _] = useState<Chain[]>(CHAINS);
```

Comment:
How come you don't need the `setChains` counterpart in the `useState`?

Reply:
yeah actually think we can remove this state entirely and just use the constant `CHAINS`